### PR TITLE
amp-image-slider P0 ABE draft

### PR DESCRIPTION
**Pull request by kevinkassimo**
Draft for amp-image-slider P0 ABE examples

### CODE REVIEW

File: src/20_Components/amp-image-slider.html [hunk 1]
```diff
@@ -0,0 +1,109 @@
+<!---{
+  "draft": true,
+  "experiments": ["amp-image-slider"]
+}--->
+<!--
+  ## Introduction
+
+  The amp-image-slider component allows comparing 2 images by moving the vertical slider bar.
```
Comment (sebastianbenz) [inline src/20_Components/amp-image-slider.html]: Can you make ` amp-image-slider` link to the official AMP Docs.

File: src/20_Components/amp-image-slider.html [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+      src="/img/canoe_900x600.jpg"
+      alt="An image about canoeing"
+      layout="fill"></amp-img>
+  </amp-image-slider>
+
+  <!--
+    You can add customizable text labels to your images, by providing extra `div`s,
+    labeled with `first` and `second` attributes, for left and right images correspondingly.
+    You can customize the style of your labels with classes.
+    If you want to place the label at exactly center of the images, other than `left` and `right` rules, you may want `translate` tricks.
```
Comment (sebastianbenz) [inline src/20_Components/amp-image-slider.html]: Can you rephrase `, you may want `translate` tricks`

File: src/20_Components/amp-image-slider.html [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+      src="/img/green_apple_1_1024x682.jpg"
+      alt="A green apple"
+      layout="fill"></amp-img>
+    <div first class="label label-left-center">Red</div>
+    <div second class="label label-right-center">Green</div>
+  </amp-image-slider>
+
+  <!--
+    Arrow hints about moving the image slider will always show by default.
+    Once user starts to interact with the slider (such as clicking and touching), the hint would fade out.
```
Comment (sebastianbenz) [inline src/20_Components/amp-image-slider.html]: s/would/will

Comment (sebastianbenz) [reply src/20_Components/amp-image-slider.html]: s/user/user starts/users start/

File: src/20_Components/amp-image-slider.html [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+    The hints would then reappear if slider leaves the viewport and later comes back again.
```
Comment (sebastianbenz) [inline src/20_Components/amp-image-slider.html]: s/would then reappear if/reappear if the

**Comment by kevinkassimo** [comment]
@sebastianbenz `amp-image-slider` has its basic functionally rolled out in 1533692711873, but there is a minor caveat that will only be rolled out in next week's canary (due to a late spec change), which would affect the demos slightly (does not change demo examples tho, just some hint reappear effects described in demo is currently missing). Do you think we should wait until everything hits production, canary, or fine to merge since it is just a draft?

**Comment by sebastianbenz** [comment]
@kevinkassimo tbh I think it's fine to publish it this way. Can you please remove the draft flag and I will merge and publish then.

**Comment by sebastianbenz** [comment]
Thanks a lot!
